i18n.schemas.js

```diff
@@ -48,6 +48,7 @@ const en = {
   'lifelines-webshop-order-applicationform-description': 'Word or text file to describe the request.',
   'lifelines-webshop-order-header': 'Order variables',
   'lifelines-webshop-order-submit-error-project-nummber': 'Please enter a project number before submitting a order.',
+  'lifelines-webshop-toggle-variables': 'Toggle All Variables',
```

Comment:
maybe go for ; 'Add all variables to cart' and 'Remove all variables from cart' toggle was not clear to me

Reply:
Updated button texts